ci/provisioning/post_provision_config_nodes_EL_8.sh

```diff
@@ -15,19 +15,12 @@ group_repo_post() {
 }
 
 distro_custom() {
-    # install avocado
-    local avocado_rpms=(python3-avocado{,-plugins-{output-html,varianter-yaml-to-mux}})
-    if [ -z "$(dnf repoquery "${avocado_rpms[@]}")" ]; then
-        avocado_rpms=()
-        pip install "avocado-framework<83.0"
-        pip install "avocado-framework-plugin-result-html<83.0"
-        pip install "avocado-framework-plugin-varianter-yaml-to-mux<83.0"
-    fi
-    dnf -y install "${avocado_rpms[@]}" clustershell
-
-    # for Launchable's pip install
-    dnf -y install python3-setuptools.noarch
+    # TODO: This code is not exiting on failure.
 
+    # Use a more recent python version for unit testing, this allows us to also test installing
+    # pydaos into virtual environments.
+    # TODO: el8 should have python3.11 available but our CI does not for some reason.
```

Comment:
@brianjmurrell not a big issue but on other wolf nodes I can yum install python3.11 but not here and I can't explain the difference.

Reply:
it looks like the CI nodes are stuck on 8.7 and python3.11 isn't available until 8.8 which explains why this isn't installable but I did think we were running 8.8 already.

Reply:
Nodes provisioned per test (i.e. Functional testing, at least) should be 8.8 now.

Reply:
Should be at 8.8 now.